cmd/earthly/subcmd/build_cmd.go

```diff
@@ -590,15 +590,6 @@ func (a *Build) ActionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs []stri
 	// information to logbusSetup. This function will be called right at the
 	// beginning of the build within earthfile2llb.
 	buildOpts.MainTargetDetailsFunc = func(d earthfile2llb.TargetDetails) error {
-		if a.cli.LogbusSetup().LogStreamerStarted() {
```

Comment:
this is also dead code since the only function that can turn this condition true is `StartLogStreamer` which is only called once in the codebase, a few lines below.